clientdb/order.go

```diff
@@ -42,11 +42,16 @@ var (
 	//
 	// path: ordersBucketKey -> orderBucket[nonce] -> orderMinUnitsMatchKey
 	orderMinUnitsMatchKey = []byte("order-min-units-match")
+
+	// orderTlvKey is a key within the order bucket for additional, tlv
+	// encoded data.
+	orderTlvKey = []byte("order-tlv")
 )
 
 type extraOrderData struct {
 	minNodeTier   order.NodeTier
 	minUnitsMatch order.SupplyUnit
+	tlvData       []byte
```

Comment:
Ah very cool, so now in the future we'll map any new fields added to the orders on the proto level to his new field?

Reply:
Also I don't think a migration is necessary, given that we're using a new key here vs modifying the existing values of an order.